backend/src/game_server.rs

```diff
@@ -155,7 +160,7 @@ impl GameServer {
         }
 
         self.broadcast(&game);
-        self.next_turn();
+        self.send_turn();
```

Comment:
first: next turn changes player and send turn. I can't see where player changes now

second: remove next_turn if you don't use it anymore

Reply:
Answer to both questions:
https://github.com/Miol-Mor/battle-game/blob/d08aaf474322d3f72cd4c57e0f9abaeb0e11ab5a/backend/src/game_server.rs#L72

Reply:
I see now!